aws-bundle/LICENSE

```diff
@@ -219,7 +219,7 @@ License: The Apache Software License, Version 2.0 - http://www.apache.org/licens
 
 --------------------------------------------------------------------------------
 
-Group: io.netty  Name: netty-buffer  Version: 4.1.112.Final
+Group: io.netty  Name: netty-buffer  Version: 4.1.115.Final
```

Comment:
can we fix the same in kafka-connect files as well in this PR?

Reply:
Yes, sure. Let me group all change in this PR.

Reply:
also in https://github.com/apache/iceberg/blob/1.8.x/gradle/libs.versions.toml#L73 we're using 4.1.117.Final btw

Reply:
Yes, but it's what I see in the aws-bundle dependencies:

```
$ ./gradlew :iceberg-aws-bundle:dependencies|grep -i netty-buffer
|         |    +--- io.netty:netty-buffer:4.1.115.Final
|         |    |    +--- io.netty:netty-buffer:4.1.115.Final (*)

```

Reply:
@jbonofre the kafka-connect files are missing the update to the netty version

Reply:
Let me double check. I might forgot to push.

Reply:
@nastra I double check and the version is OK in kafka-connect runtime distribution: netty 4.1.115.Final artifacts are in the distribution `LICENSE` corresponding to what's in the `lib` folder. So it looks good to me.

Reply:
my bad. I didn't look at the actual dependency name. I saw `netty-codec-dns` to point to `4.1.112.Final` and assumed it wasn't properly updated yet

Reply:
Yes, that's not the same artifacts. I fully understand it could be confusing 😄